taf/log.py

```diff
@@ -9,23 +10,34 @@
 logger = logging.getLogger('taf')
 logger.setLevel(taf.settings.LOG_LEVEL)
 
+
+
 if taf.settings.ENABLE_CONSOLE_LOGGING:
   console_handler = logging.StreamHandler()
   console_handler.setLevel(taf.settings.CONSOLE_LOGGING_LEVEL)
   console_handler.setFormatter(formatter)
   logger.addHandler(console_handler)
 
+
 if taf.settings.ENABLE_FILE_LOGGING:
-  file_handler = logging.FileHandler(taf.settings.LOG_FILENAME)
+  logs_location = taf.settings.LOGS_LOCATION
+  if logs_location is None:
```

Comment:
Could you extract this to a function (`_get_log_location`)? Maybe it would be better to check if platform is windows and use `Path.home() / '.taf'` in that case.